specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-06-01-preview/apimdeployment.json

```diff
@@ -589,103 +596,6 @@
         },
         "x-ms-long-running-operation": true
       }
-    },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/updatecertificate": {
-      "post": {
-        "tags": [
-          "ApiManagementService"
-        ],
-        "operationId": "ApiManagementService_UploadCertificate",
-        "description": "Upload Custom Domain SSL certificate for an API Management service. This operation will be deprecated in future releases.",
-        "x-ms-examples": {
-          "ApiManagementServiceUploadProxyCertificate": {
-            "$ref": "./examples/ApiManagementServiceUploadProxyCertificate.json"
-          }
-        },
-        "parameters": [
-          {
-            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
-          },
-          {
-            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
-          },
-          {
-            "name": "parameters",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/ApiManagementServiceUploadCertificateParameters"
-            },
-            "description": "Parameters supplied to the Upload SSL certificate for an API Management service operation."
-          },
-          {
-            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
-          },
-          {
-            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "Successfully uploaded certificate to the API Management Service.",
-            "schema": {
-              "$ref": "#/definitions/CertificateInformation"
-            }
-          }
-        }
-      }
-    },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/updatehostname": {
```

Comment:
This API appears to still be in active use. While this is a preview API, has your deprecation policy been forgiving enough? A new API version would be a cleaner way to drop these now.

Reply:
havent seen a call to this api in last 30 days in ARM. But checked the code, it is actually deprecated in next api-version 2019-01-01. Will bring it back in swagger.

---
In reply to: [261058793](https://github.com/Azure/azure-rest-api-specs/pull/5260#discussion_r261058793) [](ancestors = 261058793)